app/templates/application.hbs

```diff
@@ -1,13 +1,9 @@
-<div class="mdl-grid">
-{{#mdl-card title='Welcome to Ember.js'}}
-  <div class="mdl-card__supporting-text">
-    You have successfully installed <a href='https://github.com/truenorth/ember-material-lite' target='_blank'>ember-material-lite</a>. It's based on an awesome google project called <a href='https://github.com/google/material-design-lite' target='_blank'>Material Design Lite</a>
+{{#mdl-nav title='La Chatelaine French Bakery & Bistro' fixedHeader=true transparent=true class='layout-transparent' }}
+  {{mdl-nav-item name='Chat Now' route=''}} {{mdl-nav-item name='Sign Up' route=''}} {{mdl-nav-item name='Log In' route=''}} {{mdl-nav-item name='My Cart' route=''}}
```

Comment:
Set the `route` attribute on login to go to `admin` so that it links to the admin interface